src/ray/core_worker/core_worker.h

```diff
@@ -30,7 +30,7 @@
   RAY_CORE_WORKER_RPC_HANDLER(AssignTask, 5)                       \
   RAY_CORE_WORKER_RPC_HANDLER(PushTask, 9999)                      \
   RAY_CORE_WORKER_RPC_HANDLER(DirectActorCallArgWaitComplete, 100) \
-  RAY_CORE_WORKER_RPC_HANDLER(GetObjectStatus, 100)
+  RAY_CORE_WORKER_RPC_HANDLER(GetObjectStatus, 9999)
```

Comment:
Do we need this too?

Reply:
Well we would if this number meant anything, but it doesn't actually limit the concurrency.